styles.css

```diff
@@ -0,0 +1,202 @@
+.container {
+  background: white;
+  border-bottom-left-radius: 50px;
+}
+
```

Comment:
- [ ] Good job @naanmohammed  😄, but as the previous reviewer said, make sure you update the styling of your headline section to better match the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=FZQJqIfsVjU0ybFj-0) as is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.). Kindly fix some font-size, margin, font weigh and color 👍

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | <img width="226" alt="mobile 2 0" src="https://user-images.githubusercontent.com/57408419/219616755-cf131229-f222-4666-8d7e-1849ec48f9be.png"> | |!![mobile](https://user-images.githubusercontent.com/57408419/219616260-321914ef-bfe3-499f-a00f-281f6f1d1c2b.jpg) | |